POS/Integration/05_ConfigureGlobalSettings.md

```diff
@@ -101,18 +94,17 @@ No prerequisites to fulfill.
 4. Click the [SAVE] button in the bottom right corner of the window.   
```

Comment:
Die Felder im Fenster "Edit payment method" heißen Titel und ID. Danach in POS heißen offenbar Key und Name. Auch wenn du nachher in Klammern (ID) schreibst, war mir ehrlich gesagt unklar, was damit gemeint war. Unter ID würde ich eigentlich ein Zahl erwartet. 
Ich würde im Schritt 2 und 3 ein Beispiel geben (oder unter 3 ein Edit payment method Screenshot mit Data, z.B. Titel 
 = EC-Karte, ID =  ec). Dann ist danach der Zusammenhang deutlicher, oder?

Reply:
done